src/ace.js

```diff
@@ -66,9 +66,7 @@ exports.edit = function(el, options) {
         onResize: editor.resize.bind(editor, null)
```

Comment:
can we remove `onResize` from `env` here? I see that it's used in doc site https://github.com/ajaxorg/ace/blob/master/doc/site/js/main.js#L158
can we rely on new functionality, introduced in this PR, in that place as well, instead of manually calling onResize?

Reply:
Good point, we can remove it from main.js, but i am not sure about env.onResize, because on one hand it is a breaking change, but on other hand it is obscure api.